client/cli/src/params/network_params.rs

```diff
@@ -41,6 +42,10 @@ pub struct NetworkParams {
 	#[clap(long, value_name = "ADDR", multiple_values(true))]
 	pub reserved_nodes: Vec<MultiaddrWithPeerId>,
 
+	/// Endpoints for requesting a pre shared key.
```

Comment:
change this comment here, this might be more generic then just requesting psk. Also,  I wonder about this name... We don't really listen to this rpc, maybe `external_nodes_rpc` will be better

Reply:
ok